modules/monitoring/files/usr/local/bin/govuk_uptime_collector

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+
+# Usage: ./collect.rb <environment> <service ...>
+
+require "csv"
+require "fileutils"
+require "net/http"
+require "time"
+require "statsd"
+
+class Array
+  def each_after(n)
+    each_with_index do |elem, i|
+      yield elem if i >= n
+    end
+  end
+end
+
+class Collector
+  def initialize(service, environment)
+    @statsd = Statsd.new("127.0.0.1")
+    @service = service
```

Comment:
Not sure if this is a good suggestion or not, but service seems like such a loaded word to me that I think `service_name` might be easier to follow?